docs/understanding-the-flutterflow-ui/Projects/how-to-collaborate-on-projects.md

```diff
@@ -0,0 +1,163 @@
+2. Click on the **Invite User**. This will open a popup to enter user's email.
+3. Input their **email address** and select **Invite**.
+4. If a user isn't already a FlutterFlow user, we will send them an invite email. Their status will be shown as **Pending** until they create an account.
+5. To provide read-only access, tick the **Read Only** checkbox.
+
+:::info
+FlutterFlow subscriptions are seat based. A user added to your project will only have access to the features of their account plan.
+:::
+
+## Branching Permissions
+The owner of a project can manage permissions related to branching. Owners can control two types of branching permissions:
+
+- Editing Permissions: Determines who is allowed to edit branching permissions.
+- Merging into Main: Specifies who is allowed to merge branches into the main branch.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Correctly maintaining the aspect ratio with additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/P5MgPJuBBeaIm7dfEjqI?embed&show_copy_link=true"
+        title="fghjkl - FlutterFlow"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameBorder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+
+## Sharing a Project with an Organization
+To share a project with all team members in your organization, enable the "Share with my organization" option.
+
+![Share Project With Organization](../../../static/img/share-project-with-organization.avif)
+
+## Real-Time Collaboration
+Real-Time Collaboration is a powerful feature that allows multiple builders to work together on the same project or, rather same page and design system simultaneously. With this, all builders can see the changes being made to the page as they happen and can also make their own changes to the page without interfering with the work of others.
+
+
+This increases efficiency and productivity, as multiple builders can work on various aspects of the project or together on the same page at the same time.
+
+To enable real-time collaboration, navigate to Settings & Integrations (from the left Navigation Menu) > Project Setup > Collaboration > click Enable Updated Collaboration.
+
+![Realtime Collaboration Setup](../../../static/img/realtime-collaboration-setup.png)
+
+Now when multiple builders are on the same page, it looks like this:
+
+![Realtime Collaboration](../../../static/img/real-time-collaboration.gif)
+
+To disable this feature anytime, click the Disable Updated Collaboration button.
+
+:::caution
+- This feature is still in Beta, and unexpected issues may occur. Ensure you regularly create project versions as a backup.
```

Comment:
create project versions ---> Add TODO comment here, this should point to docs for adding versions